sdcm/nemesis.py

```diff
@@ -2148,6 +2185,26 @@ def run_deletions(self, queries, ks_cf):
 
         self.target_node.run_nodetool('flush', args=ks_cf.replace('.', ' '))
 
+    def _verify_using_timestamp_deletions(self, ks_cf: str, verification_queries: list[tuple[int, int, int]]):
+        mv_not_configured = False
+        mv_table_name = ".".join([ks_cf.split(sep=".")[0], "view_test"])
+        with self.cluster.cql_connection_patient(self.target_node, connect_timeout=300) as session:
+            # pylint: disable=invalid-name
+            for pk, ck, ts in verification_queries:
+                result = session.execute(SimpleStatement(
+                    f"SELECT pk, ck, writetime(v) FROM {ks_cf} WHERE pk = {pk} AND ck = {ck}")).one()
+                assert not result or result.writetime_v != ts, \
```

Comment:
shouldn't we use here greater comparison and verify all the rows instead of `one()`?: `result.writetime_v > ts`